doc/Improve-Performance.md

```diff
@@ -81,6 +81,6 @@ Please refer to external documentation for a more detailed explanation how to se
 
 ### Database
 
-There are scripts like [tuning-primer.sh](http://www.day32.com/MySQL/) and [mysqltuner.pl](http://mysqltuner.pl) that analyze your database server and give hints on values that could be changed.
+There are scripts like [tuning-primer.sh](https://github.com/BMDan/tuning-primer.sh) and [mysqltuner.pl](https://github.com/major/MySQLTuner-perl/blob/master/mysqltuner.pl) that analyze your database server and give hints on values that could be changed.
```

Comment:
I’d rather we keep the links to the official websites when they exist. I don’t want GitHub to become a single point of failure for all our outgoing links.

Reply:
The day32 one is 404 while the pl one redirects to the same GitHub repo today.